Vagrantfile

```diff
@@ -21,65 +21,33 @@ end
 BAZEL_VERSION = ENV['BAZEL_VERSION']
 
 $bootstrap = <<SCRIPT
-# This may be removed when the box images are based on Ubuntu 17.10+.
-sudo bash -c "cat <<EOF > /etc/apt/sources.list
-deb http://old-releases.ubuntu.com/ubuntu/ yakkety main restricted
-deb http://old-releases.ubuntu.com/ubuntu/ yakkety-updates main restricted
-deb http://old-releases.ubuntu.com/ubuntu/ yakkety universe
-deb http://old-releases.ubuntu.com/ubuntu/ yakkety-updates universe
-deb http://old-releases.ubuntu.com/ubuntu/ yakkety multiverse
-deb http://old-releases.ubuntu.com/ubuntu/ yakkety-updates multiverse
-deb http://old-releases.ubuntu.com/ubuntu/ yakkety-backports main restricted universe multiverse
-EOF
-"
+echo "----------------------------------------------------------------"
+export PATH=/home/vagrant/go/bin:/usr/local/clang/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games
 sudo service docker restart
-sudo apt-get -y update || true
-sudo apt-get -y install socat curl jq realpath pv tmux python-sphinx python-pip
-sudo pip install --upgrade pip
-sudo pip install sphinx sphinxcontrib-httpdomain sphinxcontrib-openapi
-sudo pip install yamllint
 echo 'cd ~/go/src/github.com/cilium/cilium' >> /home/vagrant/.bashrc
-export GOPATH=/home/vagrant/go
 sudo -E /usr/local/go/bin/go get github.com/cilium/go-bindata/...
 sudo -E /usr/local/go/bin/go get -u github.com/google/gops
-chown -R vagrant:vagrant $GOPATH
+sudo chown -R vagrant:vagrant /home/vagrant
 curl -SsL https://github.com/cilium/bpf-map/releases/download/v1.0/bpf-map -o bpf-map
 chmod +x bpf-map
 mv bpf-map /usr/bin
-if [[ $(command -v bazel) && "$(bazel version | grep 'label' | cut -d ' ' -f 3)" = #{BAZEL_VERSION} ]]; then
-  echo "Bazel #{BAZEL_VERSION} already installed, skipping fetch."
-else
-  wget -nv https://github.com/bazelbuild/bazel/releases/download/#{BAZEL_VERSION}/bazel-#{BAZEL_VERSION}-installer-linux-x86_64.sh
-  chmod +x bazel-#{BAZEL_VERSION}-installer-linux-x86_64.sh
-  sudo -E ./bazel-#{BAZEL_VERSION}-installer-linux-x86_64.sh
-  sudo -E mv /usr/local/bin/bazel /usr/bin
-  rm bazel-#{BAZEL_VERSION}-installer-linux-x86_64.sh
-fi
```

Comment:
This should not be removed. Upstream Envoy master builds with the newest Bazel release, which we need to sync up to every time we rebase to upstream Envoy. Rather than requiring CI VM updates while testing the rebase PRs we may need to fetch a newer version of Bazel while provisioning the VMs.

Reply:
If there is an absolute reason for not fetching Bazel during provisioning, then we should abort the provisioning if Bazel versions do not match.

Reply:
@jrajahalme bazel is already installed in the VM, do you still want to download it every time we re-provision?

Reply:
I re-read the code, you're right this can be kept.